upgrader.go

```diff
@@ -60,22 +99,15 @@ func (u *Upgrader) UpgradeListener(t transport.Transport, list manet.Listener) t
 	return l
 }
 
-// UpgradeOutbound upgrades the given outbound multiaddr-net connection into a
-// full libp2p-transport connection.
-// Deprecated: use Upgrade instead.
-func (u *Upgrader) UpgradeOutbound(ctx context.Context, t transport.Transport, maconn manet.Conn, p peer.ID) (transport.CapableConn, error) {
+func (u *upgrader) UpgradeOutbound(ctx context.Context, t transport.Transport, maconn manet.Conn, p peer.ID) (transport.CapableConn, error) {
```

Comment:
lets nuke those, the are trivial redirects.

Reply:
Done.